crud/select/compat/select.lua

```diff
@@ -79,7 +79,7 @@ local function build_select_iterator(space_name, user_conditions, opts)
     local batch_size
     if opts.batch_size == nil then
         if opts.first ~= nil and opts.first < common.DEFAULT_BATCH_SIZE then
```

Comment:
If `opts.first` is negative, `opts.first < common.DEFAULT_BATCH_SIZE` is always true.